apps/src/gamelab/animationListModule.js

```diff
@@ -44,12 +44,57 @@ export const START_LOADING_FROM_SOURCE = 'AnimationList/START_LOADING_FROM_SOURC
 export const DONE_LOADING_FROM_SOURCE = 'AnimationList/DONE_LOADING_FROM_SOURCE';
 // Args: {AnimationKey} key, {string} version
 const ON_ANIMATION_SAVED = 'AnimationList/ON_ANIMATION_SAVED';
+// Args: {AnimationKey} key, {!SerializedAnimation} props
+const SET_PENDING_ANIMATION_ADDITION = 'AnimationList/SET_PENDING_ANIMATION_ADDITION';
+// Args: none
+export const START_LOADING_ADDITION_FROM_SOURCE = 'AnimationList/START_LOADING_ADDITION_FROM_SOURCE';
+// Args: {AnimationKey} key, {Blob} blob, {String} dataURI. Version?
+export const DONE_LOADING_ADDITION_FROM_SOURCE = 'AnimationList/DONE_LOADING_ADDITION_FROM_SOURCE';
+// Args: none
+export const REMOVE_PENDING_ANIMATION_ADDITION = 'AnimationList/REMOVE_PENDING_ANIMATION_ADDITION';
+
 
 export default combineReducers({
   orderedKeys,
-  propsByKey
+  propsByKey,
+  pendingAnimationAddition
 });
 
+// pendingAnimationAddition is used for temporary storing additional
+// frames before they get added to the animation in Piskel.
+// pendingAnimationAddition gets added to animation in PiskelEditor.jsx
+function pendingAnimationAddition(state, action) {
```

Comment:
I'm considering moving this out of AnimationList, but in some ways it makes a lot of sense here, because it will be an addition to an animation in the AnimationList. Thoughts?